src/main/java/com/suse/salt/netapi/calls/modules/Locate.java

```diff
@@ -0,0 +1,182 @@
+package com.suse.salt.netapi.calls.modules;
+
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.reflect.TypeToken;
+import com.suse.salt.netapi.calls.LocalCall;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * salt.modules.locate
+ * Module for using the mlocate utilities.
+ */
+public class Locate {
+
+    private static final LocalCall<List<String>> VERSION = new LocalCall<>(
+            "locate.version",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<List<String>>() {
+            });
+
+    private static final LocalCall<List<String>> UPDATEDB = new LocalCall<>(
+            "locate.updatedb",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<List<String>>() {
+            });
+
+    private static final LocalCall<Stats> STATS = new LocalCall<>("locate.stats",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<Stats>() {
+            });
+
+    /**
+     * Locate module result object
+     */
+    public static class Stats {
+        private Long files;
```

Comment:
The reason for the boxed `Boolean` type in `Result` is to support tri-state booleans exposed in netapi, which is indeed the case in some File module methods (`True`/`False`/`None`).

However, it turns out `None` is returned only when the API is called with the `test` argument, which we don't actually expose. So, I'll post a PR to fix the `Result` class, too.